velox/common/compression/Compression.cpp

```diff
@@ -16,6 +16,7 @@
 
 #include "velox/common/compression/Compression.h"
 #include "velox/common/base/Exceptions.h"
+#include "velox/common/compression/Lz4Compression.h"
```

Comment:
like he we would protect this with a a VELOX_ENABLE_COMPRESSION_LZ4 (or similar)